packages/formdata-event/ts_src/dispatchFormdataForSubmission.ts

```diff
@@ -9,8 +9,94 @@
  * additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
 
+import {getEntries} from './Wrappers/FormData.js';
+
 export const dispatchFormdataForSubmission = (form: HTMLFormElement) => {
```

Comment:
I've been thinking about this comment for a while and agree that the flow through the polyfill needs to be documented somewhere but was hoping to find a way to split this so that the relevant chunks were in the module with their implementations. However, I haven't been able to convince myself so far that local comments can make up for seeing the flow in one shot. I think a separate markdown file might be good for this even.

Reply:
Yeah, separate markdown sounds fine to me. That's what I did for shadow parts actually: https://github.com/webcomponents/polyfills/blob/shady-parts/packages/shadycss/shadow-parts.md

Reply:
> Yeah, separate markdown sounds fine to me. That's what I did for shadow parts actually: https://github.com/webcomponents/polyfills/blob/shady-parts/packages/shadycss/shadow-parts.md

(separate PR is fine)